vagrant up fails

Hi,

I'm trying to get kong up & running for custom plugin development but am running into the following issue. I've tried killing my repo & reinstalling virtualbox but it seems to die every time at the same place (installing java).

Am I doing something wrong? The only thing I see that stands out is the 'dpkg-preconfigure: unable to re-open stdin: No such file or directory' which apparently is an expected bug in dpkg.


~/github/kong-vagrant>$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'hashicorp/precise64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'hashicorp/precise64' is up to date...
==> default: Setting the name of the VM: vagrant_kong
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 8000 (guest) => 8000 (host) (adapter 1)
    default: 8001 (guest) => 8001 (host) (adapter 1)
    default: 8443 (guest) => 8443 (host) (adapter 1)
    default: 8444 (guest) => 8444 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 4.2.0
    default: VirtualBox Version: 5.0
==> default: Mounting shared folders...
    default: /kong => /Users/chris/github/kong
    default: /vagrant => /Users/chris/github/kong-vagrant
==> default: Running provisioner: shell...
    default: Running: /var/folders/1w/nv844fcs6p5fsrjr4m1_0r8c0000gn/T/vagrant-shell20170522-3629-fim56d.sh
==> default: Installing Kong version: 0.10.2
==> default: Ign http://us.archive.ubuntu.com precise InRelease
==> default: Get:1 http://us.archive.ubuntu.com precise-updates InRelease [55.7 kB]
==> default: Get:2 http://security.ubuntu.com precise-security InRelease [55.7 kB]
==> default: Get:3 http://us.archive.ubuntu.com precise-backports InRelease [55.7 kB]
==> default: Hit http://us.archive.ubuntu.com precise Release.gpg
==> default: Get:4 http://us.archive.ubuntu.com precise-updates/main Sources [503 kB]
==> default: Get:5 http://us.archive.ubuntu.com precise-updates/restricted Sources [8,837 B]
==> default: Get:6 http://us.archive.ubuntu.com precise-updates/universe Sources [137 kB]
==> default: Get:7 http://us.archive.ubuntu.com precise-updates/multiverse Sources [10.5 kB]
==> default: Get:8 http://us.archive.ubuntu.com precise-updates/main amd64 Packages [741 kB]
==> default: Get:9 http://security.ubuntu.com precise-security/main Sources [150 kB]
==> default: Get:10 http://security.ubuntu.com precise-security/restricted Sources [4,643 B]
==> default: Get:11 http://security.ubuntu.com precise-security/universe Sources [57.9 kB]
==> default: Get:12 http://security.ubuntu.com precise-security/multiverse Sources [3,032 B]
==> default: Get:13 http://security.ubuntu.com precise-security/main amd64 Packages [375 kB]
==> default: Get:14 http://us.archive.ubuntu.com precise-updates/restricted amd64 Packages [15.4 kB]
==> default: Get:15 http://us.archive.ubuntu.com precise-updates/universe amd64 Packages [288 kB]
==> default: Get:16 http://us.archive.ubuntu.com precise-updates/multiverse amd64 Packages [17.1 kB]
==> default: Get:17 http://us.archive.ubuntu.com precise-updates/main i386 Packages [745 kB]
==> default: Get:18 http://security.ubuntu.com precise-security/restricted amd64 Packages [10.9 kB]
==> default: Get:19 http://security.ubuntu.com precise-security/universe amd64 Packages [145 kB]
==> default: Get:20 http://us.archive.ubuntu.com precise-updates/restricted i386 Packages [15.3 kB]
==> default: Get:21 http://us.archive.ubuntu.com precise-updates/universe i386 Packages [297 kB]
==> default: Get:22 http://us.archive.ubuntu.com precise-updates/multiverse i386 Packages [17.3 kB]
==> default: Get:23 http://us.archive.ubuntu.com precise-updates/main TranslationIndex [208 B]
==> default: Get:24 http://us.archive.ubuntu.com precise-updates/multiverse TranslationIndex [202 B]
==> default: Get:25 http://us.archive.ubuntu.com precise-updates/restricted TranslationIndex [202 B]
==> default: Get:26 http://us.archive.ubuntu.com precise-updates/universe TranslationIndex [205 B]
==> default: Get:27 http://us.archive.ubuntu.com precise-backports/main Sources [5,922 B]
==> default: Get:28 http://us.archive.ubuntu.com precise-backports/restricted Sources [28 B]
==> default: Get:29 http://us.archive.ubuntu.com precise-backports/universe Sources [44.2 kB]
==> default: Get:30 http://us.archive.ubuntu.com precise-backports/multiverse Sources [5,750 B]
==> default: Get:31 http://security.ubuntu.com precise-security/multiverse amd64 Packages [3,367 B]
==> default: Get:32 http://security.ubuntu.com precise-security/main i386 Packages [377 kB]
==> default: Get:33 http://us.archive.ubuntu.com precise-backports/main amd64 Packages [6,477 B]
==> default: Get:34 http://us.archive.ubuntu.com precise-backports/restricted amd64 Packages [28 B]
==> default: Get:35 http://us.archive.ubuntu.com precise-backports/universe amd64 Packages [46.3 kB]
==> default: Get:36 http://us.archive.ubuntu.com precise-backports/multiverse amd64 Packages [5,419 B]
==> default: Get:37 http://us.archive.ubuntu.com precise-backports/main i386 Packages [6,478 B]
==> default: Get:38 http://us.archive.ubuntu.com precise-backports/restricted i386 Packages [28 B]
==> default: Get:39 http://us.archive.ubuntu.com precise-backports/universe i386 Packages [46.1 kB]
==> default: Get:40 http://us.archive.ubuntu.com precise-backports/multiverse i386 Packages [5,413 B]
==> default: Get:41 http://us.archive.ubuntu.com precise-backports/main TranslationIndex [202 B]
==> default: Get:42 http://us.archive.ubuntu.com precise-backports/multiverse TranslationIndex [202 B]
==> default: Get:43 http://us.archive.ubuntu.com precise-backports/restricted TranslationIndex [193 B]
==> default: Get:44 http://us.archive.ubuntu.com precise-backports/universe TranslationIndex [205 B]
==> default: Hit http://us.archive.ubuntu.com precise Release
==> default: Get:45 http://us.archive.ubuntu.com precise-updates/main Translation-en [344 kB]
==> default: Get:46 http://us.archive.ubuntu.com precise-updates/multiverse Translation-en [10.1 kB]
==> default: Get:47 http://us.archive.ubuntu.com precise-updates/restricted Translation-en [3,686 B]
==> default: Get:48 http://us.archive.ubuntu.com precise-updates/universe Translation-en [174 kB]
==> default: Get:49 http://us.archive.ubuntu.com precise-backports/main Translation-en [5,737 B]
==> default: Get:50 http://us.archive.ubuntu.com precise-backports/multiverse Translation-en [4,852 B]
==> default: Get:51 http://us.archive.ubuntu.com precise-backports/restricted Translation-en [28 B]
==> default: Get:52 http://us.archive.ubuntu.com precise-backports/universe Translation-en [35.9 kB]
==> default: Hit http://us.archive.ubuntu.com precise/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/restricted Sources
==> default: Get:53 http://security.ubuntu.com precise-security/restricted i386 Packages [10.8 kB]
==> default: Get:54 http://security.ubuntu.com precise-security/universe i386 Packages [154 kB]
==> default: Hit http://us.archive.ubuntu.com precise/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/restricted TranslationIndex
==> default: Get:55 http://security.ubuntu.com precise-security/multiverse i386 Packages [3,537 B]
==> default: Get:56 http://security.ubuntu.com precise-security/main TranslationIndex [208 B]
==> default: Get:57 http://security.ubuntu.com precise-security/multiverse TranslationIndex [199 B]
==> default: Get:58 http://security.ubuntu.com precise-security/restricted TranslationIndex [202 B]
==> default: Get:59 http://security.ubuntu.com precise-security/universe TranslationIndex [205 B]
==> default: Hit http://us.archive.ubuntu.com precise/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Translation-en
==> default: Get:60 http://security.ubuntu.com precise-security/main Translation-en [188 kB]
==> default: Hit http://us.archive.ubuntu.com precise/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/universe Translation-en
==> default: Get:61 http://security.ubuntu.com precise-security/multiverse Translation-en [1,993 B]
==> default: Get:62 http://security.ubuntu.com precise-security/restricted Translation-en [2,802 B]
==> default: Get:63 http://security.ubuntu.com precise-security/universe Translation-en [93.2 kB]
==> default: Fetched 5,297 kB in 3s (1,756 kB/s)
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following extra packages will be installed:
==> default:   git-man libapt-pkg4.12 libcurl3 libcurl3-gnutls liberror-perl libpcre3
==> default:   libpcrecpp0 patch
==> default: Suggested packages:
==> default:   git-daemon-run git-daemon-sysvinit git-doc git-el git-arch git-cvs git-svn
==> default:   git-email git-gui gitk gitweb make-doc diffutils-doc zip
==> default: The following NEW packages will be installed:
==> default:   curl git git-man libcurl3 liberror-perl libpcre3-dev libpcrecpp0 make patch
==> default:   pkg-config unzip
==> default: The following packages will be upgraded:
==> default:   apt-transport-https libapt-pkg4.12 libcurl3-gnutls libpcre3
==> default: 4 upgraded, 11 newly installed, 0 to remove and 189 not upgraded.
==> default: Need to get 9,141 kB of archives.
==> default: After this operation, 18.1 MB of additional disk space will be used.
==> default: Get:1 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libapt-pkg4.12 amd64 0.8.16~exp12ubuntu10.27 [934 kB]
==> default: Get:2 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libpcre3 amd64 8.12-4ubuntu0.2 [149 kB]
==> default: Get:3 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libcurl3-gnutls amd64 7.22.0-3ubuntu4.17 [228 kB]
==> default: Get:4 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libcurl3 amd64 7.22.0-3ubuntu4.17 [237 kB]
==> default: Get:5 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libpcrecpp0 amd64 8.12-4ubuntu0.2 [16.2 kB]
==> default: Get:6 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main apt-transport-https amd64 0.8.16~exp12ubuntu10.27 [16.4 kB]
==> default: Get:7 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main curl amd64 7.22.0-3ubuntu4.17 [138 kB]
==> default: Get:8 http://us.archive.ubuntu.com/ubuntu/ precise/main liberror-perl all 0.17-1 [23.8 kB]
==> default: Get:9 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main git-man all 1:1.7.9.5-1ubuntu0.3 [632 kB]
==> default: Get:10 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main git amd64 1:1.7.9.5-1ubuntu0.3 [6,103 kB]
==> default: Get:11 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libpcre3-dev amd64 8.12-4ubuntu0.2 [231 kB]
==> default: Get:12 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main make amd64 3.81-8.1ubuntu1.1 [119 kB]
==> default: Get:13 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main patch amd64 2.6.1-3ubuntu0.1 [80.0 kB]
==> default: Get:14 http://us.archive.ubuntu.com/ubuntu/ precise/main pkg-config amd64 0.26-1ubuntu1 [40.9 kB]
==> default: Get:15 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main unzip amd64 6.0-4ubuntu2.5 [193 kB]
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Fetched 9,141 kB in 5s (1,785 kB/s)
==> default: (Reading database ...
==> default: 51095 files and directories currently installed.)
==> default: Preparing to replace libapt-pkg4.12 0.8.16~exp12ubuntu10 (using .../libapt-pkg4.12_0.8.16~exp12ubuntu10.27_amd64.deb) ...
==> default: Unpacking replacement libapt-pkg4.12 ...
==> default: Setting up libapt-pkg4.12 (0.8.16~exp12ubuntu10.27) ...
==> default: Processing triggers for libc-bin ...
==> default: ldconfig deferred processing now taking place
==> default: (Reading database ...
==> default: 51095 files and directories currently installed.)
==> default: Preparing to replace libpcre3 8.12-4 (using .../libpcre3_8.12-4ubuntu0.2_amd64.deb) ...
==> default: Unpacking replacement libpcre3 ...
==> default: Preparing to replace libcurl3-gnutls 7.22.0-3ubuntu4 (using .../libcurl3-gnutls_7.22.0-3ubuntu4.17_amd64.deb) ...
==> default: Unpacking replacement libcurl3-gnutls ...
==> default: Selecting previously unselected package libcurl3.
==> default: Unpacking libcurl3 (from .../libcurl3_7.22.0-3ubuntu4.17_amd64.deb) ...
==> default: Selecting previously unselected package libpcrecpp0.
==> default: Unpacking libpcrecpp0 (from .../libpcrecpp0_8.12-4ubuntu0.2_amd64.deb) ...
==> default: Preparing to replace apt-transport-https 0.8.16~exp12ubuntu10 (using .../apt-transport-https_0.8.16~exp12ubuntu10.27_amd64.deb) ...
==> default: Unpacking replacement apt-transport-https ...
==> default: Selecting previously unselected package curl.
==> default: Unpacking curl (from .../curl_7.22.0-3ubuntu4.17_amd64.deb) ...
==> default: Selecting previously unselected package liberror-perl.
==> default: Unpacking liberror-perl (from .../liberror-perl_0.17-1_all.deb) ...
==> default: Selecting previously unselected package git-man.
==> default: Unpacking git-man (from .../git-man_1%3a1.7.9.5-1ubuntu0.3_all.deb) ...
==> default: Selecting previously unselected package git.
==> default: Unpacking git (from .../git_1%3a1.7.9.5-1ubuntu0.3_amd64.deb) ...
==> default: Selecting previously unselected package libpcre3-dev.
==> default: Unpacking libpcre3-dev (from .../libpcre3-dev_8.12-4ubuntu0.2_amd64.deb) ...
==> default: Selecting previously unselected package make.
==> default: Unpacking make (from .../make_3.81-8.1ubuntu1.1_amd64.deb) ...
==> default: Selecting previously unselected package patch.
==> default: Unpacking patch (from .../patch_2.6.1-3ubuntu0.1_amd64.deb) ...
==> default: Selecting previously unselected package pkg-config.
==> default: Unpacking pkg-config (from .../pkg-config_0.26-1ubuntu1_amd64.deb) ...
==> default: Selecting previously unselected package unzip.
==> default: Unpacking unzip (from .../unzip_6.0-4ubuntu2.5_amd64.deb) ...
==> default: Processing triggers for man-db ...
==> default: Setting up libpcre3 (8.12-4ubuntu0.2) ...
==> default: Setting up libcurl3-gnutls (7.22.0-3ubuntu4.17) ...
==> default: Setting up libcurl3 (7.22.0-3ubuntu4.17) ...
==> default: Setting up libpcrecpp0 (8.12-4ubuntu0.2) ...
==> default: Setting up apt-transport-https (0.8.16~exp12ubuntu10.27) ...
==> default: Setting up curl (7.22.0-3ubuntu4.17) ...
==> default: Setting up liberror-perl (0.17-1) ...
==> default: Setting up git-man (1:1.7.9.5-1ubuntu0.3) ...
==> default: Setting up git (1:1.7.9.5-1ubuntu0.3) ...
==> default: Setting up libpcre3-dev (8.12-4ubuntu0.2) ...
==> default: Setting up make (3.81-8.1ubuntu1.1) ...
==> default: Setting up patch (2.6.1-3ubuntu0.1) ...
==> default: Setting up pkg-config (0.26-1ubuntu1) ...
==> default: Setting up unzip (6.0-4ubuntu2.5) ...
==> default: Processing triggers for libc-bin ...
==> default: ldconfig deferred processing now taking place
==> default: Hit http://us.archive.ubuntu.com precise Release.gpg
==> default: Get:1 http://us.archive.ubuntu.com precise-updates Release.gpg [198 B]
==> default: Get:2 http://us.archive.ubuntu.com precise-backports Release.gpg [198 B]
==> default: Hit http://us.archive.ubuntu.com precise Release
==> default: Get:3 http://us.archive.ubuntu.com precise-updates Release [55.4 kB]
==> default: Get:4 http://security.ubuntu.com precise-security Release.gpg [198 B]
==> default: Get:5 http://security.ubuntu.com precise-security Release [55.5 kB]
==> default: Get:6 http://us.archive.ubuntu.com precise-backports Release [55.5 kB]
==> default: Hit http://us.archive.ubuntu.com precise/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/universe TranslationIndex
==> default: Get:7 http://us.archive.ubuntu.com precise-updates/main Sources [503 kB]
==> default: Get:8 http://us.archive.ubuntu.com precise-updates/restricted Sources [8,837 B]
==> default: Get:9 http://us.archive.ubuntu.com precise-updates/universe Sources [137 kB]
==> default: Get:10 http://us.archive.ubuntu.com precise-updates/multiverse Sources [10.5 kB]
==> default: Get:11 http://us.archive.ubuntu.com precise-updates/main amd64 Packages [741 kB]
==> default: Get:12 http://security.ubuntu.com precise-security/main Sources [150 kB]
==> default: Get:13 http://security.ubuntu.com precise-security/restricted Sources [4,643 B]
==> default: Get:14 http://security.ubuntu.com precise-security/universe Sources [57.9 kB]
==> default: Get:15 http://security.ubuntu.com precise-security/multiverse Sources [3,032 B]
==> default: Get:16 http://security.ubuntu.com precise-security/main amd64 Packages [375 kB]
==> default: Get:17 http://us.archive.ubuntu.com precise-updates/restricted amd64 Packages [15.4 kB]
==> default: Get:18 http://us.archive.ubuntu.com precise-updates/universe amd64 Packages [288 kB]
==> default: Get:19 http://us.archive.ubuntu.com precise-updates/multiverse amd64 Packages [17.1 kB]
==> default: Get:20 http://us.archive.ubuntu.com precise-updates/main i386 Packages [745 kB]
==> default: Get:21 http://security.ubuntu.com precise-security/restricted amd64 Packages [10.9 kB]
==> default: Get:22 http://security.ubuntu.com precise-security/universe amd64 Packages [145 kB]
==> default: Get:23 http://security.ubuntu.com precise-security/multiverse amd64 Packages [3,367 B]
==> default: Get:24 http://security.ubuntu.com precise-security/main i386 Packages [377 kB]
==> default: Get:25 http://security.ubuntu.com precise-security/restricted i386 Packages [10.8 kB]
==> default: Get:26 http://security.ubuntu.com precise-security/universe i386 Packages [154 kB]
==> default: Get:27 http://us.archive.ubuntu.com precise-updates/restricted i386 Packages [15.3 kB]
==> default: Get:28 http://us.archive.ubuntu.com precise-updates/universe i386 Packages [297 kB]
==> default: Get:29 http://security.ubuntu.com precise-security/multiverse i386 Packages [3,537 B]
==> default: Get:30 http://security.ubuntu.com precise-security/main TranslationIndex [208 B]
==> default: Get:31 http://security.ubuntu.com precise-security/multiverse TranslationIndex [199 B]
==> default: Get:32 http://security.ubuntu.com precise-security/restricted TranslationIndex [202 B]
==> default: Get:33 http://security.ubuntu.com precise-security/universe TranslationIndex [205 B]
==> default: Get:34 http://us.archive.ubuntu.com precise-updates/multiverse i386 Packages [17.3 kB]
==> default: Get:35 http://us.archive.ubuntu.com precise-updates/main TranslationIndex [208 B]
==> default: Get:36 http://us.archive.ubuntu.com precise-updates/multiverse TranslationIndex [202 B]
==> default: Get:37 http://us.archive.ubuntu.com precise-updates/restricted TranslationIndex [202 B]
==> default: Get:38 http://us.archive.ubuntu.com precise-updates/universe TranslationIndex [205 B]
==> default: Get:39 http://us.archive.ubuntu.com precise-backports/main Sources [5,922 B]
==> default: Hit http://security.ubuntu.com precise-security/main Translation-en
==> default: Get:40 http://us.archive.ubuntu.com precise-backports/restricted Sources [28 B]
==> default: Get:41 http://us.archive.ubuntu.com precise-backports/universe Sources [44.2 kB]
==> default: Get:42 http://us.archive.ubuntu.com precise-backports/multiverse Sources [5,750 B]
==> default: Get:43 http://us.archive.ubuntu.com precise-backports/main amd64 Packages [6,477 B]
==> default: Get:44 http://us.archive.ubuntu.com precise-backports/restricted amd64 Packages [28 B]
==> default: Get:45 http://us.archive.ubuntu.com precise-backports/universe amd64 Packages [46.3 kB]
==> default: Get:46 http://us.archive.ubuntu.com precise-backports/multiverse amd64 Packages [5,419 B]
==> default: Get:47 http://us.archive.ubuntu.com precise-backports/main i386 Packages [6,478 B]
==> default: Get:48 http://us.archive.ubuntu.com precise-backports/restricted i386 Packages [28 B]
==> default: Get:49 http://us.archive.ubuntu.com precise-backports/universe i386 Packages [46.1 kB]
==> default: Get:50 http://us.archive.ubuntu.com precise-backports/multiverse i386 Packages [5,413 B]
==> default: Get:51 http://us.archive.ubuntu.com precise-backports/main TranslationIndex [202 B]
==> default: Get:52 http://us.archive.ubuntu.com precise-backports/multiverse TranslationIndex [202 B]
==> default: Hit http://security.ubuntu.com precise-security/multiverse Translation-en
==> default: Hit http://security.ubuntu.com precise-security/restricted Translation-en
==> default: Get:53 http://us.archive.ubuntu.com precise-backports/restricted TranslationIndex [193 B]
==> default: Get:54 http://us.archive.ubuntu.com precise-backports/universe TranslationIndex [205 B]
==> default: Hit http://us.archive.ubuntu.com precise/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe Translation-en
==> default: Hit http://security.ubuntu.com precise-security/universe Translation-en
==> default: Fetched 4,431 kB in 3s (1,306 kB/s)
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following extra packages will be installed:
==> default:   python-pycurl unattended-upgrades
==> default: Suggested packages:
==> default:   libcurl4-gnutls-dev python-pycurl-dbg bsd-mailx
==> default: The following NEW packages will be installed:
==> default:   python-pycurl python-software-properties software-properties-common
==> default:   unattended-upgrades
==> default: 0 upgraded, 4 newly installed, 0 to remove and 189 not upgraded.
==> default: Need to get 106 kB of archives.
==> default: After this operation, 794 kB of additional disk space will be used.
==> default: Get:1 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main unattended-upgrades all 0.76ubuntu1.3 [24.7 kB]
==> default: Get:2 http://us.archive.ubuntu.com/ubuntu/ precise/main python-pycurl amd64 7.19.0-4ubuntu3 [49.2 kB]
==> default: Get:3 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main python-software-properties all 0.82.7.7 [23.5 kB]
==> default: Get:4 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main software-properties-common all 0.82.7.7 [8,450 B]
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Fetched 106 kB in 0s (217 kB/s)
==> default: Selecting previously unselected package unattended-upgrades.
==> default: (Reading database ...
==> default: 51889 files and directories currently installed.)
==> default: Unpacking unattended-upgrades (from .../unattended-upgrades_0.76ubuntu1.3_all.deb) ...
==> default: Selecting previously unselected package python-pycurl.
==> default: Unpacking python-pycurl (from .../python-pycurl_7.19.0-4ubuntu3_amd64.deb) ...
==> default: Selecting previously unselected package python-software-properties.
==> default: Unpacking python-software-properties (from .../python-software-properties_0.82.7.7_all.deb) ...
==> default: Selecting previously unselected package software-properties-common.
==> default: Unpacking software-properties-common (from .../software-properties-common_0.82.7.7_all.deb) ...
==> default: Processing triggers for man-db ...
==> default: Processing triggers for ureadahead ...
==> default: Setting up unattended-upgrades (0.76ubuntu1.3) ...
==> default: Setting up python-pycurl (7.19.0-4ubuntu3) ...
==> default: Setting up python-software-properties (0.82.7.7) ...
==> default: Setting up software-properties-common (0.82.7.7) ...
==> default: OK
==> default: Hit http://us.archive.ubuntu.com precise Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-updates Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-backports Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise Release
==> default: Hit http://us.archive.ubuntu.com precise-updates Release
==> default: Hit http://security.ubuntu.com precise-security Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-backports Release
==> default: Hit http://security.ubuntu.com precise-security Release
==> default: Hit http://us.archive.ubuntu.com precise/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/main Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/restricted Sources
==> default: Hit http://security.ubuntu.com precise-security/universe Sources
==> default: Hit http://security.ubuntu.com precise-security/multiverse Sources
==> default: Hit http://security.ubuntu.com precise-security/main amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/restricted amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/universe amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/multiverse amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/main i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/restricted i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/main Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe Translation-en
==> default: Hit http://security.ubuntu.com precise-security/multiverse i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/main TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/multiverse TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/restricted TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/universe TranslationIndex
==> default: Get:1 https://apt.postgresql.org precise-pgdg Release.gpg [819 B]
==> default: Hit http://security.ubuntu.com precise-security/main Translation-en
==> default: Hit http://security.ubuntu.com precise-security/multiverse Translation-en
==> default: Hit http://security.ubuntu.com precise-security/restricted Translation-en
==> default: Get:2 https://apt.postgresql.org precise-pgdg Release [40.1 kB]
==> default: Hit http://security.ubuntu.com precise-security/universe Translation-en
==> default: Get:3 https://apt.postgresql.org precise-pgdg/main Sources [29.9 kB]
==> default: Get:4 https://apt.postgresql.org precise-pgdg/main amd64 Packages [74.3 kB]
==> default: Get:5 https://apt.postgresql.org precise-pgdg/main i386 Packages [74.3 kB]
==> default: Get:6 https://apt.postgresql.org precise-pgdg/main TranslationIndex [345 B]
==> default: Ign https://apt.postgresql.org precise-pgdg/main TranslationIndex
==> default: Ign https://apt.postgresql.org precise-pgdg/main Translation-en_US
==> default: Ign https://apt.postgresql.org precise-pgdg/main Translation-en
==> default: Fetched 219 kB in 4s (48.9 kB/s)
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following extra packages will be installed:
==> default:   libpq5 libsensors4 libxslt1.1 pgdg-keyring postgresql-client-9.5
==> default:   postgresql-client-common postgresql-common postgresql-contrib-9.5 ssl-cert
==> default:   sysstat
==> default: Suggested packages:
==> default:   lm-sensors locales-all postgresql-doc-9.5 libjson-perl libdbd-pg-perl
==> default:   openssl-blacklist isag
==> default: The following NEW packages will be installed:
==> default:   libpq5 libsensors4 libxslt1.1 pgdg-keyring postgresql-9.5
==> default:   postgresql-client-9.5 postgresql-client-common postgresql-common
==> default:   postgresql-contrib-9.5 ssl-cert sysstat
==> default: 0 upgraded, 11 newly installed, 0 to remove and 189 not upgraded.
==> default: Need to get 6,457 kB of archives.
==> default: After this operation, 28.6 MB of additional disk space will be used.
==> default: Get:1 http://us.archive.ubuntu.com/ubuntu/ precise/main libsensors4 amd64 1:3.3.1-2ubuntu1 [31.9 kB]
==> default: Get:2 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main libxslt1.1 amd64 1.1.26-8ubuntu1.4 [168 kB]
==> default: Get:3 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main ssl-cert all 1.0.28ubuntu0.1 [12.3 kB]
==> default: Get:4 http://us.archive.ubuntu.com/ubuntu/ precise/main sysstat amd64 10.0.3-1 [303 kB]
==> default: Get:5 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main libpq5 amd64 9.6.3-1.pgdg12.4+1 [128 kB]
==> default: Get:6 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main pgdg-keyring all 2017.1 [10.1 kB]
==> default: Get:7 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main postgresql-client-common all 182.pgdg12.4+1 [81.7 kB]
==> default: Get:8 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main postgresql-client-9.5 amd64 9.5.7-1.pgdg12.4+1 [1,182 kB]
==> default: Get:9 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main postgresql-common all 182.pgdg12.4+1 [236 kB]
==> default: Get:10 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main postgresql-9.5 amd64 9.5.7-1.pgdg12.4+1 [3,866 kB]
==> default: Get:11 https://apt.postgresql.org/pub/repos/apt/ precise-pgdg/main postgresql-contrib-9.5 amd64 9.5.7-1.pgdg12.4+1 [438 kB]
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Fetched 6,457 kB in 4s (1,426 kB/s)
==> default: Selecting previously unselected package libpq5.
==> default: (Reading database ...
==> default: 51976 files and directories currently installed.)
==> default: Unpacking libpq5 (from .../libpq5_9.6.3-1.pgdg12.4+1_amd64.deb) ...
==> default: Selecting previously unselected package libsensors4.
==> default: Unpacking libsensors4 (from .../libsensors4_1%3a3.3.1-2ubuntu1_amd64.deb) ...
==> default: Selecting previously unselected package libxslt1.1.
==> default: Unpacking libxslt1.1 (from .../libxslt1.1_1.1.26-8ubuntu1.4_amd64.deb) ...
==> default: Selecting previously unselected package pgdg-keyring.
==> default: Unpacking pgdg-keyring (from .../pgdg-keyring_2017.1_all.deb) ...
==> default: Selecting previously unselected package postgresql-client-common.
==> default: Unpacking postgresql-client-common (from .../postgresql-client-common_182.pgdg12.4+1_all.deb) ...
==> default: Selecting previously unselected package postgresql-client-9.5.
==> default: Unpacking postgresql-client-9.5 (from .../postgresql-client-9.5_9.5.7-1.pgdg12.4+1_amd64.deb) ...
==> default: Selecting previously unselected package ssl-cert.
==> default: Unpacking ssl-cert (from .../ssl-cert_1.0.28ubuntu0.1_all.deb) ...
==> default: Selecting previously unselected package postgresql-common.
==> default: Unpacking postgresql-common (from .../postgresql-common_182.pgdg12.4+1_all.deb) ...
==> default: Adding 'diversion of /usr/bin/pg_config to /usr/bin/pg_config.libpq-dev by postgresql-common'
==> default: Selecting previously unselected package postgresql-9.5.
==> default: Unpacking postgresql-9.5 (from .../postgresql-9.5_9.5.7-1.pgdg12.4+1_amd64.deb) ...
==> default: Selecting previously unselected package postgresql-contrib-9.5.
==> default: Unpacking postgresql-contrib-9.5 (from .../postgresql-contrib-9.5_9.5.7-1.pgdg12.4+1_amd64.deb) ...
==> default: Selecting previously unselected package sysstat.
==> default: Unpacking sysstat (from .../sysstat_10.0.3-1_amd64.deb) ...
==> default: Processing triggers for man-db ...
==> default: Processing triggers for ureadahead ...
==> default: Setting up libpq5 (9.6.3-1.pgdg12.4+1) ...
==> default: Setting up libsensors4 (1:3.3.1-2ubuntu1) ...
==> default: Setting up libxslt1.1 (1.1.26-8ubuntu1.4) ...
==> default: Setting up pgdg-keyring (2017.1) ...
==> default: Removing apt.postgresql.org key from trusted.gpg:
==> default: OK
==> default: Setting up postgresql-client-common (182.pgdg12.4+1) ...
==> default: Setting up postgresql-client-9.5 (9.5.7-1.pgdg12.4+1) ...
==> default: update-alternatives:
==> default: using /usr/share/postgresql/9.5/man/man1/psql.1.gz to provide /usr/share/man/man1/psql.1.gz (psql.1.gz) in auto mode.
==> default: Setting up ssl-cert (1.0.28ubuntu0.1) ...
==> default: Setting up postgresql-common (182.pgdg12.4+1) ...
==> default: Adding user postgres to group ssl-cert
==> default: Creating config file /etc/postgresql-common/createcluster.conf with new version
==> default: Creating config file /etc/logrotate.d/postgresql-common with new version
==> default: Building PostgreSQL dictionaries from installed myspell/hunspell packages...
==> default: Removing obsolete dictionary files:
==> default:  * No PostgreSQL clusters exist; see "man pg_createcluster"
==> default: Setting up postgresql-9.5 (9.5.7-1.pgdg12.4+1) ...
==> default: Creating new PostgreSQL cluster 9.5/main ...
==> default: /usr/lib/postgresql/9.5/bin/initdb -D /var/lib/postgresql/9.5/main --auth-local peer --auth-host md5
==> default: The files belonging to this database system will be owned by user "postgres".
==> default: This user must also own the server process.
==> default: The database cluster will be initialized with locale "en_US".
==> default: The default database encoding has accordingly been set to "LATIN1".
==> default: The default text search configuration will be set to "english".
==> default: Data page checksums are disabled.
==> default: fixing permissions on existing directory /var/lib/postgresql/9.5/main ...
==> default: ok
==> default: creating subdirectories ...
==> default: ok
==> default: selecting default max_connections ...
==> default: 100
==> default: selecting default shared_buffers ...
==> default: 128MB
==> default: selecting dynamic shared memory implementation ...
==> default: posix
==> default: creating configuration files ...
==> default: ok
==> default: creating template1 database in /var/lib/postgresql/9.5/main/base/1 ...
==> default: ok
==> default: initializing pg_authid ...
==> default: ok
==> default: initializing dependencies ...
==> default: ok
==> default: creating system views ...
==> default: ok
==> default: loading system objects' descriptions ...
==> default: ok
==> default: creating collations ...
==> default: ok
==> default: creating conversions ...
==> default: ok
==> default: creating dictionaries ...
==> default: ok
==> default: setting privileges on built-in objects ...
==> default: ok
==> default: creating information schema ...
==> default: ok
==> default: loading PL/pgSQL server-side language ...
==> default: ok
==> default: vacuuming database template1 ...
==> default: ok
==> default: copying template1 to template0 ...
==> default: ok
==> default: copying template1 to postgres ...
==> default: ok
==> default: syncing data to disk ...
==> default: ok
==> default:
==> default: Success. You can now start the database server using:
==> default:
==> default:     /usr/lib/postgresql/9.5/bin/pg_ctl -D /var/lib/postgresql/9.5/main -l logfile start
==> default: Ver Cluster Port Status Owner    Data directory               Log file
==> default: 9.5 main    5432 down   postgres /var/lib/postgresql/9.5/main /var/log/postgresql/postgresql-9.5-main.log
==> default: update-alternatives:
==> default: using /usr/share/postgresql/9.5/man/man1/postmaster.1.gz to provide /usr/share/man/man1/postmaster.1.gz (postmaster.1.gz) in auto mode.
==> default:  * Starting PostgreSQL 9.5 database server
==> default:    ...done.
==> default: Setting up postgresql-contrib-9.5 (9.5.7-1.pgdg12.4+1) ...
==> default: Setting up sysstat (10.0.3-1) ...
==> default: Creating config file /etc/default/sysstat with new version
==> default: update-alternatives:
==> default: using /usr/bin/sar.sysstat to provide /usr/bin/sar (sar) in auto mode.
==> default: Processing triggers for libc-bin ...
==> default: ldconfig deferred processing now taking place
==> default:  * Restarting PostgreSQL 9.5 database server
==> default:    ...done.
==> default: Expanded display is on.
==> default: CREATE ROLE
==> default: CREATE DATABASE
==> default: CREATE DATABASE
==> default: Hit http://us.archive.ubuntu.com precise Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-updates Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-backports Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise Release
==> default: Hit http://us.archive.ubuntu.com precise-updates Release
==> default: Hit http://security.ubuntu.com precise-security Release.gpg
==> default: Hit http://us.archive.ubuntu.com precise-backports Release
==> default: Hit http://security.ubuntu.com precise-security Release
==> default: Hit http://us.archive.ubuntu.com precise/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/restricted i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/main Sources
==> default: Hit http://us.archive.ubuntu.com precise/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/main Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise-updates/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/restricted Sources
==> default: Hit http://security.ubuntu.com precise-security/universe Sources
==> default: Hit http://security.ubuntu.com precise-security/multiverse Sources
==> default: Hit http://security.ubuntu.com precise-security/main amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/restricted amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/universe amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/multiverse amd64 Packages
==> default: Hit http://security.ubuntu.com precise-security/main i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/restricted i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/main Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse Sources
==> default: Hit http://us.archive.ubuntu.com precise-backports/main amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com precise-backports/main TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-updates/universe Translation-en
==> default: Hit http://security.ubuntu.com precise-security/multiverse i386 Packages
==> default: Hit http://security.ubuntu.com precise-security/main TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/multiverse TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/restricted TranslationIndex
==> default: Hit http://security.ubuntu.com precise-security/universe TranslationIndex
==> default: Hit http://us.archive.ubuntu.com precise-backports/main Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com precise-backports/universe Translation-en
==> default: Hit http://security.ubuntu.com precise-security/main Translation-en
==> default: Hit http://security.ubuntu.com precise-security/multiverse Translation-en
==> default: Hit http://security.ubuntu.com precise-security/restricted Translation-en
==> default: Hit https://apt.postgresql.org precise-pgdg Release.gpg
==> default: Hit http://security.ubuntu.com precise-security/universe Translation-en
==> default: Hit https://apt.postgresql.org precise-pgdg Release
==> default: Hit https://apt.postgresql.org precise-pgdg/main Sources
==> default: Hit https://apt.postgresql.org precise-pgdg/main amd64 Packages
==> default: Hit https://apt.postgresql.org precise-pgdg/main i386 Packages
==> default: Get:1 https://apt.postgresql.org precise-pgdg/main TranslationIndex [345 B]
==> default: Ign https://apt.postgresql.org precise-pgdg/main TranslationIndex
==> default: Ign https://apt.postgresql.org precise-pgdg/main Translation-en_US
==> default: Ign https://apt.postgresql.org precise-pgdg/main Translation-en
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following NEW packages will be installed:
==> default:   redis-server
==> default: 0 upgraded, 1 newly installed, 0 to remove and 189 not upgraded.
==> default: Need to get 204 kB of archives.
==> default: After this operation, 523 kB of additional disk space will be used.
==> default: Get:1 http://us.archive.ubuntu.com/ubuntu/ precise/universe redis-server amd64 2:2.2.12-1build1 [204 kB]
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Fetched 204 kB in 0s (360 kB/s)
==> default: Selecting previously unselected package redis-server.
==> default: (Reading database ...
==> default: 52887 files and directories currently installed.)
==> default: Unpacking redis-server (from .../redis-server_2%3a2.2.12-1build1_amd64.deb) ...
==> default: Processing triggers for ureadahead ...
==> default: ureadahead will be reprofiled on next reboot
==> default: Processing triggers for man-db ...
==> default: Setting up redis-server (2:2.2.12-1build1) ...
==> default: Starting redis-server:
==> default: redis-server.
==> default: Fetching and installing java...
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

I didn't make it very far, though.

> vagrant@precise64:/kong$ make dev
> Missing dependencies for kong 0.10.2-0:
>    lua-cassandra == 1.2.2 (1.2.1-0 installed)
> 
> kong 0.10.2-0 depends on lua-cassandra == 1.2.2 (1.2.1-0 installed)
> Installing https://luarocks.org/lua-cassandra-1.2.2-0.rockspec
> Cloning into 'lua-cassandra'...
> fatal: unable to connect to github.com:
> github.com[0: 192.30.253.112]: errno=Connection refused
> github.com[1: 192.30.253.113]: errno=Connection refused
> 
> 
> Error: Failed installing dependency: https://luarocks.org/lua-cassandra-1.2.2-0.rockspec - Failed cloning git repository.
> make: *** [install] Error 1
> vagrant@precise64:/kong$

--- a/provision.sh
+++ b/provision.sh
@@ -61,10 +61,10 @@ sudo chown vagrant /var/log/redis/redis-server.log
 # Install java runtime (Cassandra dependency)
 echo Fetching and installing java...
 sudo mkdir -p /usr/lib/jvm
-sudo wget -q -O /tmp/jre-linux-x64.tar.gz --no-cookies --no-check-certificate --header 'Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie' http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jre-8u60-linux-x64.tar.gz
+sudo wget -q -O /tmp/jre-linux-x64.tar.gz --no-cookies --no-check-certificate --header 'Cookie: oraclelicense=accept-securebackup-cookie' http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jre-8u131-linux-x64.tar.gz
 sudo tar zxvf /tmp/jre-linux-x64.tar.gz -C /usr/lib/jvm
-sudo update-alternatives --install '/usr/bin/java' 'java' '/usr/lib/jvm/jre1.8.0_60/bin/java' 1
-sudo update-alternatives --set java /usr/lib/jvm/jre1.8.0_60/bin/java
+sudo update-alternatives --install '/usr/bin/java' 'java' '/usr/lib/jvm/jre1.8.0_131/bin/java' 1
+sudo update-alternatives --set java /usr/lib/jvm/jre1.8.0_131/bin/java
 
 # install cassandra
 echo Fetching and installing Cassandra...

